backend/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.8'
```

Comment:
Not sure if's because I need to update my `docker-compose` but it complains about the version, can we set this to `3` instead?